src/Composer/Repository/PathRepository.php

```diff
@@ -91,14 +98,14 @@ class PathRepository extends ArrayRepository implements ConfigurableRepositoryIn
     private $process;
 
     /**
-     * @var array{symlink?: bool, relative?: bool, versions?: array<string, string>}
+     * @var array{symlink?: bool, reference?: string, relative: bool, versions?: array<string, string>}
```

Comment:
This seems to be a regression causing the PHPStan warning below 

```suggestion
     * @var array{symlink?: bool, reference?: string, relative?: bool, versions?: array<string, string>}
```